gmso/core/subtopology.py

```diff
@@ -70,17 +70,23 @@ def parent(self, parent):
             )
         self._parent = _validate_parent(parent)
 
-    def add_site(self, site):
+    def add_site(self, site, update_types=True):
         """Add a site to this sub-topology
 
         This method adds a site to the sub-topology.
         If the sub-topology has a parent, the site will
-        also be added to the parent topology.
+        also be added to the parent topology. If the
+        update_types parameter is set to true (default
+        behavior), this method will also check if there
+        is an gmso.AtomType associated with the site and
+        it to the sub-topology's AtomTypes collection.
 
         Parameters
         ----------
         site : gmso.Atom
             The site to be added to this sub-topology
+        update_types : (bool), default=True
```

Comment:
This is just nitpicking, but can you remove the parentheses (so just `update_types : bool, default=True`) just so we are consistent with other method.